modules/ROOT/pages/dataweave-formats-xml.adoc

```diff
@@ -23,7 +23,26 @@ To understand the parsing strategies that DataWeave readers and writers can appl
 [[format_xml_cdata]]
 == CData Custom Type
 
-`CData` is a custom DataWeave data type for XML that is used to identify a Character DATA (CData) block. The `CData` type tells the XML writer to wrap content inside a `CData` block or to check for an input string inside a `CData` block. In DataWeave, `CData` inherits from the type `String`.
+`CData` is a custom DataWeave data type for XML that is used to identify a character data (CDATA) block. The `CData` type makes the XML writer wrap content inside a `CDATA` block or to check for an input string inside a `CDATA` block. In DataWeave, `CData` inherits from the type `String`.
+
+== Doctype Custom Type
+
+`DocType` is a custom DataWeave data type for XML that is used to identify a doctype declaration.
+
+== Read and Write DTDs
+
+_Introduced in DataWeave 2.5.0. Supported by Mule 4.5.0 and later._
+
+DataWeave can read and write doctype declarations in XML files.
+
+To read a doctype declarations, set the xref:dataweave-system-properties.adoc[system property] `com.mulesoft.dw.xml_reader.parseDtd`.
+
+During the reading phase, DataWeave parses the doctype declarations and stores the content as `metadata` of the root element.
+
+You can use the xref:dataweave-selectors.adoc[metadata selector] to access it.
+
+NOTE::
+DTDs are disabled by default even if their declarations are read or written. For details, see the xref:dataweave-formats-xml.adoc#properties[XML reader property] `supportDtd`.
```

Comment:
TODO: Check the anchor and link. This is the xref:dataweave-formats-xml.adoc file. Might need to adjust in the source.

Reply:
reader_properties is the anchor in this file

Reply:
fixed in PR https://github.com/mulesoft/data-weave/pull/2042/files 
and works as expected